src/game.js

```diff
@@ -0,0 +1,38 @@
+class Game {
+  constructor () {
+    this.frame = 0;
+    this.gameScore = new Array(10);
+    this.frameClass = new Frame();
+    this.scoreClass = new Score();
+  }
+
+  roll (roll1, roll2 = null, roll3 = null) {
+    if (this.frame !== 9 && roll3 !== null) {
+      return console.log('Unable to input 3 rolls unless in tenth frame');
+    }
+    if (this.frame !== 9 && roll1 + roll2 > 10) {
+      return console.log('Rolls must not exceed a total of ten');
+    }
+    let bonus = this.frameClass.calcBonus(roll1, roll2);
+    if (this.frame === 9) {
+      var score = this.scoreClass.calculateFrame10(roll1, roll2, roll3);
+    } else {
+      var score = this.scoreClass.calculate(roll1, roll2, bonus);
+    }
+    this.storeScore(score[0], score[1], score[2]);
+    this.frame += 1;
```

Comment:
JS shorthand you can use ++ instead of += 1

I like the control flow here. 

The only thing that confused me was the use of 9 as the 10th frame, but guess this revolves around indexing.